<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QStackedWidget" name="mianStackWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>下一个</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>300</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>折线图</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>柱状图</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>饼状图</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>雷达图</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>旭日图</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>17</width>
               <height>139</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="frameShape">
           <enum>QFrame::WinPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWebEngineView" name="web_1"/>
          <widget class="QWebEngineView" name="web_2"/>
          <widget class="QWebEngineView" name="web_3"/>
          <widget class="QWebEngineView" name="web_4"/>
          <widget class="QWebEngineView" name="web_5"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QToolButton" name="openButton">
                <property name="text">
                 <string>打开表</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="newButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>新建表</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="deleteButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>删除表</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_6" native="true">
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QTabWidget" name="tabWidget">
                <property name="tabPosition">
                 <enum>QTabWidget::North</enum>
                </property>
                <property name="tabShape">
                 <enum>QTabWidget::Triangular</enum>
                </property>
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <property name="elideMode">
                 <enum>Qt::ElideNone</enum>
                </property>
                <widget class="QWidget" name="tab">
                 <attribute name="title">
                  <string>表1</string>
                 </attribute>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QTableView" name="tableView_1">
                    <attribute name="horizontalHeaderMinimumSectionSize">
                     <number>25</number>
                    </attribute>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="tab_2">
                 <attribute name="title">
                  <string>表2</string>
                 </attribute>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QTableView" name="tableView_2"/>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="tab_3">
                 <attribute name="title">
                  <string>表3</string>
                 </attribute>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QTableView" name="tableView_3"/>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="tab_4">
                 <attribute name="title">
                  <string>表4</string>
                 </attribute>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QTableView" name="tableView_4"/>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="tab_5">
                 <attribute name="title">
                  <string>表5</string>
                 </attribute>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QTableView" name="tableView_5">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="autoScrollMargin">
                     <number>16</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_4" native="true">
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="lineEdit"/>
                 </item>
                 <item row="1" column="0">
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>177</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="0" colspan="4">
                  <widget class="QWidget" name="widget_5" native="true">
                   <layout class="QGridLayout" name="gridLayout">
                    <item row="0" column="1">
                     <widget class="QPushButton" name="addRecordButton">
                      <property name="text">
                       <string>添加</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="2" colspan="2">
                     <spacer name="horizontalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="1" column="5">
                     <spacer name="horizontalSpacer_6">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>205</width>
                        <height>17</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="0" column="0">
                     <spacer name="horizontalSpacer_3">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>206</width>
                        <height>17</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="1" column="1">
                     <widget class="QPushButton" name="undoButton">
                      <property name="text">
                       <string>撤销</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="4">
                     <widget class="QPushButton" name="deleteRecordButton">
                      <property name="text">
                       <string>删除</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="5">
                     <spacer name="horizontalSpacer_4">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>205</width>
                        <height>17</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="1" column="0">
                     <spacer name="horizontalSpacer_7">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>206</width>
                        <height>17</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="1" column="4">
                     <widget class="QPushButton" name="submitButton">
                      <property name="text">
                       <string>提交修改</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="2" colspan="2">
                     <spacer name="horizontalSpacer_5">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QPushButton" name="queryButton">
                   <property name="text">
                    <string>查询</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="3">
                  <spacer name="horizontalSpacer_8">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>194</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>打开</string>
    </property>
    <addaction name="actionDataBase"/>
    <addaction name="separator"/>
    <addaction name="actionDataView"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDataBase">
   <property name="text">
    <string>数据库</string>
   </property>
  </action>
  <action name="actionDataView">
   <property name="text">
    <string>数据视图</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
